src/services/refactors/moveToFile.ts

```diff
@@ -151,6 +152,9 @@ import {
 import {
     registerRefactor,
 } from "../refactorProvider";
+import {
+    isBlockLike,

```

Comment:
This indicates that we should be moving this helper somewhere else; refactors typically don't import each other.

Reply:
And in fact, you can see that it reorders the refactorings offered because they get registered in a different order now.